addons/website_sale/static/src/js/editor/snippets.options.js

```diff
@@ -1,8 +1,15 @@
 import options from "@web_editor/js/editor/snippets.options";
 
 options.registry.MegaMenuLayout = options.registry.MegaMenuLayout.extend({
-    _fetchEcommerceCategories() {
-        return this.containerEl.classList.contains('fetchEcomCategories');
+
+    /**
+     * Initialize fetchEcomCategories to the value of the button.
```

Comment:
We initiate it to the last known state, as the button doesn't have any value yet. IMO, `Initialize fetchEcomCategories.` is enough.
```suggestion
     * Initialize fetchEcomCategories.
```